aospy/internal_names.py

```diff
@@ -42,8 +42,8 @@
 AVERAGE_T2_STR = 'average_T2'
 TIME_WEIGHTS_STR = 'time_weights'
 NV_STR = 'nv'
-AVG_START_DATE_STR = 'avg_start_date'
-AVG_END_DATE_STR = 'avg_end_date'
+AVG_START_DATE_STR = 'data_start_date'
+AVG_END_DATE_STR = 'data_end_date'
```

Comment:
These attributes serve an important internal purpose, so we shouldn't change what they represent (they fix #72).  I'll add new ones to track the actual subset range.

I've renamed these ```'raw_data_start_date'``` and ```'raw_data_end_date'```, and I've called the new attributes ```'subset_start_date'``` and ```'subset_end_date'```.  Let me know if that's a little clearer.